package NYT.NHive.NProto;

import "yt/core/misc/guid.proto";

////////////////////////////////////////////////////////////////////////////////

message TReqAcknowledgeMessages
{
    required NYT.NProto.TGuid cell_guid = 1;
    required int32 last_incoming_message_id = 2;
}

////////////////////////////////////////////////////////////////////////////////

